pkg/accounting/rollup/rollup.go

```diff
@@ -174,11 +171,6 @@ func (r *Service) RollupBW(ctx context.Context, lastRollup time.Time, rollupStat
 			r.logger.Info("delete order type")
 		}
 	}
-	//remove the latest day (which we cannot know is complete), then push to DB
-	latestTally = time.Date(latestTally.Year(), latestTally.Month(), latestTally.Day(), 0, 0, 0, 0, latestTally.Location())
-	delete(rollupStats, latestTally)
-	if len(rollupStats) == 0 {
-		r.logger.Info("Rollup only found data for today")
-	}
+
```

Comment:
i thought you had made a change to the tests to catch this bug? I thought you added .Location() somewhere to the tests?